.scalafix.conf

```diff
@@ -1,17 +1,21 @@
 rules = [
-  NoAutoTupling
-  RemoveUnused
+  SortImports
+  Disable
+  DisableSyntax
+  ExplicitResultTypes
   LeakingImplicitClassVal
+  MissingFinal
+  NoAutoTupling
   NoValInForComprehension
   ProcedureSyntax
-  DisableSyntax
-  ExplicitResultTypes
-  Disable
+  RemoveUnused
 ]
 
-ExplicitResultTypes {
-  unsafeShortenNames = true
-}
+SortImports.blocks = [
```

Comment:
this is how IDEA does it, so I would keep it that way -- we don't want Scalafix and IDEA fight each other